app/bases/knative/config/summaries.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: summaries
+  namespace: scanners
+spec:
+  schedule: "0 5 * * *"
```

Comment:
How can we guarantee that this is always after the scans have been ran?

Reply:
Updated to reflect the schedule agreed upon in our discussion